rma/wizard/stock_picking_return.py

```diff
@@ -129,6 +124,10 @@ def create_returns(self):
                 )
             vals_list = self._prepare_rma_vals_list()
             rmas = self.env["rma"].create(vals_list)
+            proc_group_vals = rmas._prepare_procurement_group_vals()
```

Comment:
This must be called with only one records:

```suggestion
            proc_group_vals = rmas[:1]._prepare_procurement_group_vals()
```